torchci/lib/bot/cliParser.ts

```diff
@@ -187,7 +187,7 @@ parser.add_argument("-h", "--help", {
   action: "store_true",
 });
 
-const botCommandPattern = new RegExp(/^ *@pytorch(merge|)bot.*$/m);
+const botCommandPattern = new RegExp(/^ *@pytorch(merge|)bot .*$/m);
```

Comment:
Consider using a generic whitespace delimiter, which will work with tabs and other separators as well https://stackoverflow.com/a/47763239/21539

```suggestion
const botCommandPattern = new RegExp(/^ *@pytorch(merge|)bot\s.*$/m);
```

Reply:
Andrey already commented that, but we don't want line breaks and other, we want literal space.